components/fxa-client/src/lib.rs

```diff
@@ -202,4 +252,10 @@ mod tests {
                 .to_string()
         );
     }
+
+    #[test]
+    fn test_deserialize_push_message() {
+        let json = "{\"version\":1,\"command\":\"fxaccounts:command_received\",\"data\":{\"command\":\"send-tab-recv\",\"index\":1,\"sender\":\"bobo\",\"url\":\"https://mozilla.org\"}}";
+        let _: PushPayload = serde_json::from_str(&json).unwrap();
+    }
```

Comment:
You're very likely to receive push messages other than send-tab, e.g. for "new device connected".  It's probably worth adding a test that they are successfully ignored, rather than causing things to blow up.